homeassistant/components/roborock/sensor.py

```diff
@@ -122,6 +127,14 @@ class RoborockSensorDescription(
         entity_category=EntityCategory.DIAGNOSTIC,
         options=RoborockErrorCode.keys(),
     ),
+    RoborockSensorDescription(
+        key="battery",
+        translation_key="vacuum_battery",
```

Comment:
```suggestion
```

Reply:
You can use device class translations here

Reply:
Good call, thanks